lib/GameConstants.cpp

```diff
@@ -215,49 +215,43 @@ std::string FactionID::encode(const si32 index)
 
 std::ostream & operator<<(std::ostream & os, const EActionType actionType)
 {
-	static const std::map<EActionType, std::string> actionTypeToString =
-	{
-		{EActionType::END_TACTIC_PHASE, "End tactic phase"},
-		{EActionType::INVALID, "Invalid"},
-		{EActionType::NO_ACTION, "No action"},
-		{EActionType::HERO_SPELL, "Hero spell"},
-		{EActionType::WALK, "Walk"},
-		{EActionType::DEFEND, "Defend"},
-		{EActionType::RETREAT, "Retreat"},
-		{EActionType::SURRENDER, "Surrender"},
-		{EActionType::WALK_AND_ATTACK, "Walk and attack"},
-		{EActionType::SHOOT, "Shoot"},
-		{EActionType::WAIT, "Wait"},
-		{EActionType::CATAPULT, "Catapult"},
-		{EActionType::MONSTER_SPELL, "Monster spell"},
-		{EActionType::BAD_MORALE, "Bad morale"},
-		{EActionType::STACK_HEAL, "Stack heal"},
-		{EActionType::DAEMON_SUMMONING, "Daemon summoning"}
-	};
+	static const std::map<EActionType, std::string> actionTypeToString = { { EActionType::END_TACTIC_PHASE, "End tactic phase" },
+																		   { EActionType::INVALID, "Invalid" },

```

Comment:
this is a strange format

Reply:
Yeah. Somehow fixed itself after second pass...